test/paymentmethods.collective.to.collective.test.js

```diff
@@ -0,0 +1,438 @@
+      // Create stripe connected account to host of collective 1
+      await store.stripeConnectedAccount(collective1.HostCollectiveId);
+
+      // Add credit card to Collective 1
+      const collectivePaymentMethod = await models.PaymentMethod.create({
+        name: '4242',
+        service: 'stripe',
+        type: 'creditcard',
+        token: 'tok_123456781234567812345678',
+        CollectiveId: collective1.HostCollectiveId,
+        monthlyLimitPerMember: 10000
+      });
+
+      //finding opencollective payment method for Collective 1
+      openCollectivePaymentMethod = await models.PaymentMethod.findOne({ where: {type: 'collective', CollectiveId: collective1.id}});
+
+      // get Balance given the created user
+      const ocPaymentMethodBalance = await openCollectivePaymentMethod.getBalanceForUser(user1);
+
+      // Setting up order
+      const order = {
+        fromCollective: { id: collective1.id },
+        collective: { id: collective3.id },
+        paymentMethod: { uuid: collectivePaymentMethod.uuid },
+        totalAmount: ocPaymentMethodBalance.amount/10,
+      }
+      // Executing queries
+      const res = await utils.graphqlQuery(createOrderQuery, { order }, user1);
+
+      // Then there should be Errors
+      expect(res.errors).to.exist;
+    });/** END OF "Collective 1 fails to send money to Collective 3 because Collective 3\'s Host does not have a credit card" */
+
+    it('Collective 1 sends money to Collective 3 through different hosts(no platform fees, but still has stripe and host fees)', async () => {
+      // Add User 1 as Collective 1 Admin
+      await models.Member.create({
+        CreatedByUserId: user1.id,
+        MemberCollectiveId: user1.CollectiveId,
+        CollectiveId: collective1.id,
+        role: 'ADMIN'
+      });
+
+      // Create stripe connected account to host of collective 1
+      await store.stripeConnectedAccount(collective1.HostCollectiveId);
+      // Add credit card to Collective 1
+      await models.PaymentMethod.create({
+        name: '4242',
+        service: 'stripe',
+        type: 'creditcard',
+        token: 'tok_123456781234567812345678',
+        CollectiveId: collective1.HostCollectiveId,
+        monthlyLimitPerMember: 10000
+      });
+
+      // Create stripe connected account to host of collective 3
+      await store.stripeConnectedAccount(collective3.HostCollectiveId);
+      // Add credit card to Collective 3
+      await models.PaymentMethod.create({
+        name: '4343',
+        service: 'stripe',
+        type: 'creditcard',
+        token: 'tok_123456781234567812345678',
+        CollectiveId: collective3.HostCollectiveId,
+        monthlyLimitPerMember: 10000
+      });
+
+      //finding opencollective payment method for Collective 1
+      openCollectivePaymentMethod = await models.PaymentMethod.findOne({ where: {type: 'collective', CollectiveId: collective1.id}});
+
+      // Setting up order with amount less than the credit card monthly limit
```

Comment:
What happens when the amount is higher than that number? 
We should check for the error message in that case.
We should also make sure that if the error happens as part of the cron job (for recurring donations), the proper error message is sent by email (and cc us?)